projects/plugins/jetpack/extensions/blocks/ai-assistant/transforms/index.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+/**
+ * Internal dependencies
+ */
+import { blockName } from '..';
+import { EXTENDED_BLOCKS, isPossibleToExtendBlock } from '../extensions/ai-assistant';
+import { PromptItemProps } from '../lib/prompt';
+
+const transformFromCoreEmbed = {
```

Comment:
```suggestion
const transformFromCore = {
```

Maybe we can drop the `Embed` here since we are not aiming at the core embed block.